content/en/_index.html

```diff
@@ -18,7 +18,9 @@ <h2>Get started with Brainhack cloud</h2>
 		<p>
 			<img src = "logo.png" style="width:200px; padding:10px" alt="brainhack logo">
 			<br>
-			<img src="oracle.png" style="width:200px; padding:10px">
+			<a href="https://www.oracle.com/research/?source=:pp:sn:::::RC_WWMK211117P00113:Evergreen_General_WWMK211117P00113_Sponshorship">
```

Comment:
Is the `?source... ` part of the URL necessary?

Reply:
not sure. I merged it now to test.